group.py

```diff
@@ -2,4 +2,12 @@
 
 # Your code to go here...
 
-my_group =
+my_group = {"Jill": {"age": 26, "job": "biologist", "relations": {"Zalika": "friend", "John": "partner"}},"Zalika": {"age": 28, "job": "artist", "relations": {"Jill": "friend"}}, "John": {"age": 27, "job": "writer", "relations": {"Jill": "partner"}}, "Nash": {"age": 34, "job": "chef", "relations": {"John": "cousin", "Zalika": "landlord"}}}
+
+print(max({name: room['age'] for name, room in my_group.items()}.values()))
```

Comment:
you can work straight on a list comprehension here. Also I don't think these are `room`s that you're looking at. I'd go for a different name for each value in the group
```suggestion
print(max([room['age'] for room in my_group.values()]))
```